lib/FCM/Util/ConfigReader.pm

```diff
@@ -310,7 +310,6 @@ sub _parse_fcm2_var {
                 $E->CONFIG_VAR_UNDEF, $state_ref->{ctx}, "undef($symbol)",
             );
         }
-        $substitute ||= q{};
```

Comment:
I had always thought that `||=` was restricted to `undef` - but this thread suggests otherwise (http://qs343.pair.com/~monkperl/index.pl?node_id=980650). It is not super obvious in perlop.